code/datums/ai_laws.dm

```diff
@@ -402,7 +402,7 @@ var/global/list/mommi_laws = list(
 	)
 
 /datum/ai_laws/gravekeeper
-	name = "Elder's Instructions"
+	name = "GraveKEEPER"
```

Comment:
I changed all the names to be descriptive of the actual lawset. I don't know where else these names are used, and a quick search through the codebase didn't get me any results (but it's difficult to search for this properly). For this lawset in particular it doesn't really matter though, because it's not used in the UI, so if you want to keep the name as it was I have no qualms about it.